oci/private/image.sh.tpl

```diff
@@ -122,6 +122,12 @@ for ARG in "$@"; do
             FIXED_ARGS+=("--annotation=$in")
           done <"${ARG#--annotations-file=}"
           ;;
+        (--cmd-file=*)
+          FIXED_ARGS+=("--cmd=$(<"${ARG#--cmd-file=}")")
```

Comment:
I am surprised that `--cmd` supports a file instead of an array of strings.

Reply:
Hi, Thank you for review: 

It doesn't take a file: `$(<file)` is reading the file. Please: see: 
https://unix.stackexchange.com/questions/102894/what-does-the-expression-dir-file-mean-in-bash

It's pattern used above. 

----

Please take a look at crane docs: 

https://github.com/google/go-containerregistry/blob/main/cmd/crane/doc/crane_mutate.md

```
      --cmd strings                 New cmd to set
      --entrypoint strings       New entrypoint to set
```
(plural strings)

AFAIR sending multiple --cmd flags caused the crane to misbehave (fail or just use the last flag).

Reply:
Let me (tomorrow) expand the test to cover multiple flags scenario...  either way worth having it and I might be wrong.